ct/komga.sh

```diff
@@ -21,7 +21,7 @@ check_container_storage
 check_container_resources
 echo -e "Loading..."
 APP="Komga"
-var_disk="2"
+var_disk="4"
```

Comment:
If your changing this back to 4 do we need to update the komga.json to reflect that?

Reply:
Yes, 2 is a little bit less, the storage check says no :D